NEWS

```diff
@@ -7,6 +7,9 @@ older versions.
 Current git version
 -------------------
 
```

Comment:
Missed that in the previous PR: but should we note the new default color theme?

Reply:
Not sure whether it's relevant because it's only a new default setting and not a new feature/functionality.

Reply:
I don't know either. Not sure what the usual workflow for users actually is, when checking out a new hlwm release. I usually do a vimdiff of my autostart and the default one in shared/ and pick the cherries :)

It's not that important to me, just wanted to see what's your opinion about it :)